File: 프로그래머스/unrated/181844. 배열의 원소 삭제하기/배열의 원소 삭제하기.js
// delete_list.indexOf(arr[i])

function solution(arr, delete_list) {
  let ans = []

  for (let i = 0; i < arr.length; i++) {
    if (!(delete_list.indexOf(arr[i]) +1)) {
      ans.push(arr[i])
    }
  }
  
  return ans
}

File: 프로그래머스/0/181844. 배열의 원소 삭제하기/배열의 원소 삭제하기.java
import java.util.ArrayList;
import java.util.List;

class Solution {
  public int[] solution(int[] arr, int[] delete_list) {
    List<Integer> result = new ArrayList<>();

    for (int num : arr) {
      result.add(num);
    }

    for (int num : delete_list) {
      result.remove((Integer) num);
    }

    return result.stream().mapToInt(i -> i).toArray();
  }
}